java/java-analysis-impl/src/com/intellij/codeInspection/localCanBeFinal/LocalCanBeFinal.java

```diff
@@ -351,6 +317,45 @@ public void visitForStatement(@NotNull PsiForStatement statement) {
     return problems;
   }
 
+  @Nullable
+  public static ControlFlow getControlFlow(final PsiCodeBlock body) {
```

Comment:
Please document the public method. What kind of control flow is returned? How it differs from existing methods in `ControlFlowFactory`?